Assets/FullSerializer/Source/Converters/fsDateConverter.cs

```diff
@@ -27,31 +27,45 @@ public class fsDateConverter : fsConverter {
         public override fsResult TrySerialize(object instance, out fsData serialized, Type storageType) {
             if (instance is DateTime) {
                 var dateTime = (DateTime)instance;
-                serialized = new fsData(dateTime.ToString(DateTimeFormatString));
+                if(Serializer.Config.SerializeDateTimeAsInteger) {
+                    serialized = new fsData(dateTime.Ticks);
+                } else {
+                    serialized = new fsData(dateTime.ToString(DateTimeFormatString));
+                    return fsResult.Success;
+                }
                 return fsResult.Success;
             }
 
             if (instance is DateTimeOffset) {
                 var dateTimeOffset = (DateTimeOffset)instance;
-                serialized = new fsData(dateTimeOffset.ToString(DateTimeOffsetFormatString));
+                serialized = new fsData(dateTimeOffset.Ticks);
```

Comment:
Is this change intentional? The fsConfig comment implies that this should still be serialized as a string.